ProjectTemplates/templates/BlazorServerWeb-CSharp/Services/MicrosoftGraphServiceExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Graph;
+using Microsoft.Identity.Web;
+
+namespace BlazorServerWeb_CSharp.Services
+{
+    public static class MicrosoftGraphServiceExtensions
+    {
+        /// <summary>
+        /// Adds the Microsoft Graph client as a singleton.
+        /// </summary>
+        /// <param name="services">Service collection.</param>
+        /// <param name="initialScopes">Initial scopes.</param>
+        /// <param name="graphBaseUrl">Base URL for Microsoft graph. This can be
+        /// changed for instance for applications running in national clouds</param>
+        public static void AddMicrosoftGraph(this IServiceCollection services, 
+                                             IEnumerable<string> initialScopes,
+                                             string graphBaseUrl = "https://graph.microsoft.com/v1.0")
```

Comment:
this one doesn't need to be beta?

Reply:
No, the templates use beta because there is a picture, but the graph team would prefer 1.0